app/assets/javascripts/components/menu_item.js.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var MenuItem = React.createClass({
+
+  navigate: function(hash) {
+    window.location.hash = hash;
+  },
+
+  render: function() {
+    var classStyle = "menu-item";
+    if (this.props.hash === "share-facebook" || this.props.hash === "get-started") {
+      classStyle += " first-item";
+    }
+    return <div className={classStyle} onClick={this.navigate.bind(this, this.props.hash)}>{this.props.children}</div>;
```

Comment:
- 80 column max, please. Especially inside JSX blocks.

- Try not to put too much logic inside JSX braces. `Function.bind` is a bit of an antipattern in most cases.

- Use whitespace to your advantage:

```
return (
  <div className={classStyle} onClick={this.navigate}>
);
```